services/frontend-v3/package.json

```diff
@@ -18,6 +18,7 @@
     "react": "^16.12.0",
     "react-app-rewired": "^2.1.5",
     "react-dom": "^16.12.0",
+    "react-highlight-words": "^0.16.0",
```

Comment:
is it possible to avoid importing this? I want to minimize the number of packages we use to avoid future support issues.

Reply:
Yeah, we need it for the searching, where it will highlight the letters/numbers that the admin searched for in a column. I think that is the last new technology that I will need for the admin pages.